src/utils/mod.rs

```diff
@@ -3,13 +3,60 @@
 pub mod fs;
 mod string;
 use errors::Error;
+use regex::Regex;
 
 use pulldown_cmark::{html, Event, Options, Parser, Tag, OPTION_ENABLE_FOOTNOTES,
                      OPTION_ENABLE_TABLES};
+
 use std::borrow::Cow;
 
 pub use self::string::{RangeArgument, take_lines};
 
+pub fn remove_html_tags<'a>(text: &'a str) -> Cow<'a, str> {
+    let regex = Regex::new(r"<[^>]*?>").unwrap();
```

Comment:
Is it a good idea to use a regex here instead of something like a HTML sanitizer? I'm not 100% familiar with how the `pulldown-cmark` markdown renderer deals with angle brackets, but if we're not careful when calling it, we could accidentally clobber the wrong stuff (e.g. an innocent paragraph like "if \`x < 0\` and \`x > -5\`, blah blah blah").

Of course, that approach could also be overkill and 95% of the time the regex will work just fine...

Reply:
Using a Regex on HTML does feel bad, but I didn't want to pull in a whole sanitizer. Angle brackets already have to be escaped in HTML blocks so it could clobber some data, but I blame the user. Worst-case scenario is that some pathological text inside of an HTML tag isn't in the search index.